# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-x86-mce-noavail-errlog.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.11/arch/x86/kernel/cpu/mce/core.c.vanilla	2024-10-09 15:19:56.426047699 +0200
+++ linux-6.11/arch/x86/kernel/cpu/mce/core.c	2024-10-09 15:20:10.146047629 +0200
@@ -2808,7 +2808,7 @@
 
 	if (!mce_available(&boot_cpu_data)) {
 		err = -EIO;
-		goto err_out;
+		return err;
 	}
 
 	if (!zalloc_cpumask_var(&mce_device_initialized, GFP_KERNEL)) {
